grid-cli/docs/kubernetes.md

```diff
@@ -53,6 +53,96 @@ $ tfcmd deploy kubernetes -n kube --ssh ~/.ssh/id_rsa.pub --master-node 14 --wor
 11:43AM INF worker0 mycelium ip: 423:16f5:ca74:b600:ff0f:bb5a:6036:56f6
 ```
 
+## Update
+
+### Add workers
+
+```bash
+tfcmd apdate kubernetes add [flags]
+```
+
+### Required Flags
+
+- name: name for the master node deployment also used for canceling the cluster deployment. must be unique.
+- ssh: path to public ssh key to set in the master node.
+
+### Optional Flags
+
+- workers-number: number of workers to be added to the cluster (default = 1)
+- workers-nodes: array of nodes ids workers should be deployed on and the remaining unassigned workers will be randomly assigned to nodes that meet the specifications.
+- workers-farm: farm id workers should be deployed on, if set choose available node from farm that fits master specs (default 1). note: workers-node and workers-farm flags cannot be set both.
+- workers-ipv4: assign public ipv4 for each worker node (default false)
+- workers-ipv6: assign public ipv6 for each worker node (default false)
+- workers-ygg: assign yggdrasil ip for each worker node (default true)
+- workers-mycelium: assign mycelium ip for each worker node (default true)
+- workers-cpu: number of cpu units for each worker node (default 1).
+- workers-memory: memory size for each worker node in GB (default 1).
+- workers-disk: disk size in GB for each worker node (default 2).
+
+> **Note:** optional flags other than workers-number and workers-nodes will be applied only if the cluster has no worker, other wise they will be ignored and the options used on cluster creation will be applied.
+
+Example:
+
+```console
+➜  grid-cli git:(development-support-updating-k8s-cluster-grid-cli) ✗ go run main.go deploy kubernetes --name test --ssh ~/.ssh/id_rsa.pub --workers-number 1 --master-memory 2 --mycelium
```

Comment:
remove `grid-cli git:(development-support-updating-k8s-cluster-grid-cli) ✗ go run main.go` and add `tfcmd .. `

check others